.scripts/setup_vscode.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
+
+require "json"
+
+config = `curl -s 'https://raw.githubusercontent.com/bigbinary/wheel/main/.vscode/extensions.json'`
+extensions = JSON.parse(config)["recommendations"]
+
+extensions.each do |extension|
+  output = `code --install-extension #{extension}`
```

Comment:
We need to check whether `code` or `code-insiders` is a valid executable command, as is it's available under the `PATH` or not. Then only we should continue with this statement.

If either `code` or `code-insiders` is missing, then print the message `Couldn't find an executable for VSCode. Please ensure that the code or code-insiders command is available in your PATH env`.